rma/models/rma.py

```diff
@@ -0,0 +1,903 @@
+# Copyright 2020 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+from odoo.tests import Form
+from odoo.tools import html2plaintext
+import odoo.addons.decimal_precision as dp
+from odoo.addons.stock.models.stock_move import PROCUREMENT_PRIORITIES
+
+
+class Rma(models.Model):
+    _name = "rma"
+    _description = "RMA"
+    _order = "priority,date desc"
```

Comment:
Otherwise the order is a little bit weird :)
```suggestion
    _order = "date desc, priority"
```